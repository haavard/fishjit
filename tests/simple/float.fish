v
 (-9)/3 = -3
\09-3,nao
 9/(-3) = -3
\903-,nao
 (-10)/3 = -3.33333
\0a-3,nao
 10/2 = 5
\a2,nao;
