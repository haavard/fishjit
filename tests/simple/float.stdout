-3
-3
-3.333333333333333
5
